x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_signal.test.ts

```diff
@@ -124,31 +136,39 @@ describe('buildSignal', () => {
       },
       status: 'open',
       rule: {
+        author: [],
+        id: '7a7065d7-6e8b-4aae-8d20-c93613dec9f9',
+        created_at: new Date(ANCHOR_DATE).toISOString(),
+        updated_at: new Date(ANCHOR_DATE).toISOString(),
         created_by: 'elastic',
-        description: 'Detecting root and admin users',
+        description: 'some description',
         enabled: true,
-        false_positives: [],
+        false_positives: ['false positive 1', 'false positive 2'],
         from: 'now-6m',
-        id: '04128c15-0d1b-4716-a4c5-46997ac7f3bd',
         immutable: false,
-        index: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
-        interval: '5m',
-        risk_score: 50,
-        rule_id: 'rule-1',
-        language: 'kuery',
-        max_signals: 100,
-        name: 'Detect Root/Admin Users',
-        output_index: '.siem-signals',
+        name: 'Query with a rule id',
         query: 'user.name: root or user.name: admin',
-        references: ['http://www.example.com', 'https://ww.example.com'],
+        references: ['test 1', 'test 2'],
```

Comment:
ℹ️  Fun fact:, on the UI, it validates references to be valid http/https but for the REST interface we don't validate that it has to be a URL. During discussions I really was curious about if users would want both the front and backend to accept just strings but everyone said nope, URL's are good.

<img width="988" alt="Screen Shot 2020-09-22 at 7 47 54 PM" src="https://user-images.githubusercontent.com/1151048/93955296-a6e9da80-fd0c-11ea-9410-a41a87e64fdc.png">

So that's why these tests contain URL's